pkg/api/validate/openshiftcluster_validatedynamic.go

```diff
@@ -124,6 +124,33 @@ func (dv *openShiftClusterSlimDynamicValidator) Dynamic(ctx context.Context) err
 	return nil
 }
 
+// Validate permissions on a Vnet
+func (dv *openShiftClusterSlimDynamicValidator) ValidateVnetPermissions(ctx context.Context) error {
```

Comment:
This has no differences from than the current `Dynamic` method. Can we use this directly? Currently, it exposes only 1 validation.